nightwatch.conf.js

```diff
@@ -0,0 +1,17 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+const nightwatchConfig = require('terra-toolkit/lib/nightwatch/nightwatch.config.js').default;
+const webpackConfig = require('./packages/terra-site/webpack.config');
+const getPackageTestDirectories = require('terra-toolkit/lib/nightwatch/setup-helper.js').getPackageTestDirectories;
+
+let srcFolders;
+const isRepoTest = !process.cwd().includes('/packages/');
+
+if (isRepoTest) {
+  srcFolders = getPackageTestDirectories('lerna.json');
+} else {
+  srcFolders = 'tests/nightwatch/';
```

Comment:
& here should prob update to `path.join('tests', 'nightwatch');`

Reply:
👍